PROJECT

```diff
@@ -1,34 +1,28 @@
-version: "2"
 domain: x-k8s.io
 repo: sigs.k8s.io/cluster-api
 resources:
 - group: cluster
-  version: v1alpha3
   kind: Cluster
-- group: cluster
   version: v1alpha3
-  kind: Machine
 - group: cluster
+  kind: Machine
   version: v1alpha3
-  kind: MachineSet
 - group: cluster
+  kind: MachineSet
   version: v1alpha3
-  kind: MachineDeployment
 - group: cluster
+  kind: MachineDeployment
   version: v1alpha3
-  kind: MachinePool
 - group: cluster
-  version: v1alpha4
   kind: Cluster
-- group: cluster
   version: v1alpha4
-  kind: Machine
 - group: cluster
+  kind: Machine
   version: v1alpha4
-  kind: MachineSet
 - group: cluster
+  kind: MachineSet
   version: v1alpha4
-  kind: MachineDeployment
 - group: cluster
+  kind: MachineDeployment
   version: v1alpha4
-  kind: MachinePool
+version: "2"
```

Comment:
Was this change done manually?

Reply:
no the re-ordering was auto-generated, but I removed MachinePools from the file which weren't supposed to be in here